app/src/main/res/layout/dashboard_header.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<TextView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This file seems like it can easily be inlined

Reply:
This is a header for a `recyclerView` should i create the view programmatically?

Reply:
Ah! My bad. This implementation is correct